relayer/metrics/metricsService.go

```diff
@@ -89,11 +89,16 @@ func prepareArrayForProject(projectData map[string]map[string]*AggregatedMetric,
 	var toSendData []RelayAnalyticsDTO
 	for chainKey, chainData := range projectData {
 		for apiTypekey, apiTypeData := range chainData {
+			var averageLatency int64
+			if apiTypeData.RelaysCount > 0 {
```

Comment:
if all relays are failed, you still have division by zero

Reply:
yes i misstyped the  if there